lib/commanded/event_store/adapters/extreme/event_publisher.ex

```diff
@@ -35,13 +34,11 @@ defmodule Commanded.EventStore.Adapters.Extreme.EventPublisher do
 
   @impl GenServer
   def handle_cast(:subscribe, state) do
-    %State{event_store: event_store, stream_name: stream_name} = state
+    %State{spear_conn_name: spear_conn_name, stream_name: stream_name} = state
 
-    {:ok, subscription} = Extreme.subscribe_to(event_store, self(), stream_name)
+    {:ok, subscription_ref} = Spear.subscribe(spear_conn_name, self(), stream_name, raw?: true)
```

Comment:
I am not sure `Extreme.subsribe_to` and `Spear.subscribe` are equivalent.
In `Commanded` `subsribe_to` is about persistent streams and `subscribe` is about transient.
If `Extreme` has the same convention, then we should use `Spear.connect_to_persistent_subscription/5`

Reply:
`Extreme` does not follow the same naming convention as commanded, it is closer to spear. 

i.e. [`Extreme.subscribe_to`](https://hexdocs.pm/extreme/Extreme.html#c:subscribe_to/3) equals [`Spear.subscribe`](https://hexdocs.pm/spear/Spear.html#subscribe/4), and [`Extreme.connect_to_persistent_subscription`](https://hexdocs.pm/extreme/Extreme.html#c:connect_to_persistent_subscription/4) equals 
[`Spear.connect_to_persistent_subscription`](https://hexdocs.pm/spear/Spear.html#connect_to_persistent_subscription/5)